app/src/main/res/layout/fragment_setting.xml

```diff
@@ -140,12 +140,28 @@
         android:ems="10"
         android:hint="*********"
```

Comment:
Can you change this hint to "········"? Interpuncts are the standard password character.

Reply:
I'm going to challenge this because the "········" is ugly to me 

<img width="367" alt="screen shot 2018-03-06 at 12 59 04 pm" src="https://user-images.githubusercontent.com/14812977/37058106-3e3ee284-213e-11e8-86ae-02dbd3dcdb81.png">
<img width="368" alt="screen shot 2018-03-06 at 12 58 44 pm" src="https://user-images.githubusercontent.com/14812977/37058107-3e549e80-213e-11e8-9f94-c4cc317ddb38.png">

Reply:
That's not the way Android passwords look by default.
![screenshot_20180306-142846](https://user-images.githubusercontent.com/14947114/37062301-e0262a6a-214a-11e8-8a51-de557542c1f8.png)

Reply:
This is a hint. Not a user typing.

I still don't believe we should revert to the default look on the basis that it looks uglier than what we have now. I'll make the changes if the team thinks otherwise.